OPENCLOUD.cmake

```diff
@@ -1,8 +1,15 @@
-set( APPLICATION_NAME       "OpenCloud Desktop" )
-set( APPLICATION_SHORTNAME  "OpenCloud" )
-set( APPLICATION_EXECUTABLE "opencloud" )
-set( APPLICATION_DOMAIN     "opencloud.eu" )
-set( APPLICATION_VENDOR     "OpenCloud" )
+option(CANARY_BUILD "Use a standalone profile" OFF)
+
+if (CANARY_BUILD)
+    set(THEME_SUFFIX " Canary")
```

Comment:
Can we use a different term than "Canary"? Something like "Head" or "git" for example?